src/main/java/com/hms_networks/americas/sc/thingworx/TWConnectorConsts.java

```diff
@@ -43,6 +43,68 @@
    * take extra processing time, especially in installations with large string tag counts.
    */
   public static final boolean QUEUE_DATA_STRING_HISTORY_ENABLED_DEFAULT = false;
+  /**
```

Comment:
For this commit is this behavior we want to support in most connectors? I see a lot of this functionality (and consts) applicable to a library. Maybe a diagnostic tag manager library that you configure with diagnostic tag objects (name, desc, server, type, handler).

Reply:
That is my plan/hope. Just did not have time to implement it now.

Reply:
In addition to this functionality, I also wanted to implement an abstract system for the queue that can potentially modify its behavior or add functionality with additional time trackers, etc.